stdlib/3/builtins.pyi

```diff
@@ -9,6 +9,7 @@ from typing import (
 from abc import abstractmethod, ABCMeta
 from types import TracebackType
 import sys
+from pathlib import Path
```

Comment:
This has to be conditional on the Python version too. I'd just move the import inside the version_info check you added.